iaso/migrations/0246_add_feature_flags.py

```diff
@@ -0,0 +1,59 @@
+from django.db import migrations
+
+feature_flags = [
+    {
+        "code": "CHECK_POSITION_FOR_FORMS",
+        "name": "Mobile: Enforce users are within reach of the Org Unit before starting a form.",
+    },
+    {"code": "ENTITY", "name": "Mobile: Show entities screen (requires authentification)"},
```

Comment:
When I set it requires authentication, I didn't mean to add it in the description but set the corresponding boolean to true